### DISCUSSION TITLE
Kwikset 910 not loading config

### DISCUSSION THREAD

**Discussion by slewis-bd** [zwave-js/zwave-js#2381] [Q&A] [answered]
I have two Kwikset 910 locks. One was added without any issues, all the config parameters loaded and the values populated as expected. When I attempted to add the second lock, which is physically closer to my controller, it doesn't load all the config parameters and the values are all undefined. Both locks seem to be identified correctly, 144-1-1 (0x0090-0x0001-0x0001) and are running the same firmware version.

Things I've tried.
- Removed and re-added in secure and in-secure mode
- Tried adding with a different USB controller
- Did a factory reset on the other USB controller, blew away all config files and repeated the process
- Excluded from other USB, removed the batteries from the lock for about 5 minutes, then did a secure inclusion with my normal USB controller
- Clicked the re-interview button a bunch of times during the above steps
- Reset the lock to factory
- Replaced the batteries

I'm not too sure what else to try. Any suggestions, tips or ideas?

**Comment by AlCalzone** [comment]
Without a log file on level debug I can only guess.

**Reply by slewis-bd** [reply]
Do you want the log file during the inclusion process at DEBUG level or silly?

**Comment by slewis-bd** [comment]
Here is a debug log file during a re-interview. I will see about getting a log file of the inclusion process later today/tonight.
[interview_debug.zip](https://github.com/zwave-js/node-zwave-js/files/6327428/interview_debug.zip)

**Reply by AlCalzone** [reply]
No need to include again. I'll check the log later

**Reply by AlCalzone** [reply]
Wrong logfile, sorry... https://zwave-js.github.io/zwavejs2mqtt/#/troubleshooting/bug_report?id=export-logs
=> 1. (the zwave-js one)

**Comment by slewis-bd** [comment]
Sorry for the delay. Here are the other logs during the interview process. [interview_debug.zip](https://github.com/zwave-js/node-zwave-js/files/6330402/interview_debug.zip)

**Comment by AlCalzone** [comment, marked as answer]
```
2021-04-18T01:34:46.396Z CNTRLR « [Node 039] node info received
                                  supported CCs:
                                  · Manufacturer Specific
                                  · Version
                                  · Security
                                  controlled CCs:

...

2021-04-18T01:34:46.553Z CNTRLR « [Node 039] received secure commands
                                  supported CCs:
                                  · Manufacturer Specific
                                  · Version
                                  · Security
                                  · Anti-Theft
                                  controlled CCs:
2021-04-18T01:34:46.553Z CNTRLR   [Node 039] The node is included securely.
```

This looks like the Anti-Theft functionality is active on this lock. I'm not sure how it was enabled, but you'll have to disable it before you can include it in zwave-js.

**Reply by slewis-bd** [reply]
Thanks! I was able to connect my old Vivint system and reset it to factory, then add the lock back to it and then disable the Anti-Theft feature, then remove and add it to zwavejs.
